### PR TITLE
Change text to clarify missing entries

### LINKED ISSUE #909: Recommend small rewrite to error message
**Is your feature request related to a problem? Please describe.**
Replace "Transitively used dependencies that should be declared directly as indicated:"

**Describe the solution you'd like**
Use text like "This project requires all transitive dependencies to be declared directly.  Add these to your build.gradle:"

**Additional context**
Basically be a bit more explicit about why there was a failure, and how to address.  I think this would help non-native speakers as well.

### PR DESCRIPTION
**Pull request by pforhan**
fixes #909

### CODE REVIEW

**Review by ZacSweers — APPROVED** [review]

File: src/main/kotlin/com/autonomousapps/internal/advice/ProjectHealthConsoleReportBuilder.kt [hunk 1]
```diff
@@ -48,7 +48,8 @@ internal class ProjectHealthConsoleReportBuilder(
 
       if (addAdvice.isNotEmpty()) {
         maybeAppendTwoLines()
-        appendReproducibleNewLine("Transitively used dependencies that should be declared directly as indicated:")
+        appendReproducibleNewLine("This project requires all transitive dependencies to be declared directly. " +
+          "Add these to your build.gradle:")
```
Comment (autonomousapps) [inline src/main/kotlin/com/autonomousapps/internal/advice/ProjectHealthConsoleReportBuilder.kt:52]: What about simply:
```
These transitive dependencies should be declared directly:
```

Comment (pforhan) [reply src/main/kotlin/com/autonomousapps/internal/advice/ProjectHealthConsoleReportBuilder.kt:52]: I think that's a good step.  Should it include the path the the build file or is that obvious from context?

Comment (autonomousapps) [reply src/main/kotlin/com/autonomousapps/internal/advice/ProjectHealthConsoleReportBuilder.kt:52]: No other advice includes a path to the build file, and in fact that path is technically dynamic so you shouldn't hardcode it. So, I'd rather not mention it at all. I will assume for now that people know where to declare their dependencies.

Comment (pforhan) [reply src/main/kotlin/com/autonomousapps/internal/advice/ProjectHealthConsoleReportBuilder.kt]: updated

**Review by autonomousapps — APPROVED** [review]
LGTM. Will merge when tests pass.

**Pull request merged**